src/App/Components/Notifications/Notifications.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/forbid-prop-types */
+import React, { Component } from 'react';
+import isEmpty from 'lodash/isEmpty';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '../../../Redux/Actions/notifications';
+
+const {
+  get, snooze, getConfigs, getProfile
+} = actions;
+
+export class Notifications extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      notifications: [],
+      isSnoozed: true
+    };
+  }
+
+  componentDidMount() {
+    const token = localStorage.getItem('token');
```

Comment:
@nkpremices I noticed you and @MCFrank16 are accessing the token directly from local storage in your components. This isn't a good practice because:

1. It means your client-side state doesn't have a single source of truth which is what Redux solves for.
2. The token in localStorage can be messed with by any user if they so wish, so let your server be the authority on user authentication, not the token.

What you should do is to set a currentUser object - containing details of the currently logged-in user -  in the redux store and use that everywhere you're currently accessing the token in localStorage.

If you have questions about this, let me know.

Reply:
We'll do it the right way as planned @MemunaHaruna